src/__tests__/parser.ts

```diff
@@ -1023,6 +1023,16 @@ describe('parser', () => {
       assert.equal(parsed.displayName, 'StatelessDisplayName');
     });
 
+    it('should be taken from stateless component `displayName` property (using named export and function expression)', () => {
```

Comment:
> Could you give me an example of a file that would fail? I am surprised we don't have it in our test suite.
> 
> Did these multiple function expressions in the single file ever work?

@pvasek 

added a test to showcase how existing behavior is broken - it's not really related to exporting single or multiple components (which you really shouldn't be doing in the first place 😛 )

rather just that if you use traditional function expression instead of arrow expression the `computeComponentName` is no longer working because of the additional  `flowNodeNameEscapedText` check